modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -62,6 +62,45 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
       $behavior['az_display_settings']['card_width_xs'] = $this->configuration['card_width_xs'];
     }
 
+    // Getting the card width checking field.
+    if (!empty($this->configuration['card_width_field'])) {
+      $card_width_field = $row->getSourceProperty($this->configuration['card_width_field']);
+      $card_count_field = $row->getSourceProperty($this->configuration['card_count_field']);
+
+      // If the card width field value is Full Width, adding the conditions.
+      if (!empty($card_width_field[0]['value']) &&
+        $card_width_field[0]['value'] === 'full_width') {
+
+        $card_width = [
+          '1' => 'col-md-12 col-lg-12',
+          '2' => 'col-md-6 col-lg-6',
+          '3' => 'col-md-4 col-lg-4',
+          '4' => 'col-md-3 col-lg-3',
+        ];
+
+        $card_width_sm = [
+          '1' => 'col-sm-12',
+          '2' => 'col-sm-6',
+          '3' => 'col-sm-4',
+          '4' => 'col-sm-3',
+        ];
+        $card_count = count($card_count_field);
+        $behavior['card_width'] = $card_width[$card_count];
+        $behavior['az_display_settings']['card_width_sm'] = $card_width_sm[$card_count];
+      }
+    }
+
+    if (!empty($row->getSourceProperty('view_mode'))) {
+      $card_style_map = [
+        'default' => 'card card-borderless',
+        'full' => 'card card-borderless',
+        'token' => 'card card-borderless',
```

Comment:
These view modes (default, full, token) aren't selectable options in QS1. Do they have to be mapped? If so, they should map to the default settings in QS2 (bordered cards).

Reply:
Done